packages/app/src/App.vue

```diff
@@ -34,7 +46,23 @@ export default defineComponent({
 </script>
 
 <style lang="css">
-.app {
+.custom-shape-divider-top-1611822391 {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  overflow: hidden;
+  line-height: 0;
+}
+
+.custom-shape-divider-top-1611822391 svg {
+  position: relative;
+  display: block;
+  width: calc(140% + 1.3px);
```

Comment:
maybe we should change that later. I heard calc is kind of bad habit

Reply:
why should it be a bad habit? its easier to see whats calculated, but at the end it could also be 140%

Reply:
removed in https://github.com/project-leek/project-leek/pull/321/commits/e4b4104301cdab99aefb7e957656adb94217d1ab